Documentation/internals/security-identities.rst

```diff
@@ -53,8 +62,9 @@ Overall, the following represents the different ranges:
 
 ::
 
-   0x00000001 - 0x000000FF (1           to 2^8  - 1)        => reserved identities
-   0x00000100 - 0x0000FFFF (2^8         to 2^16 - 1)        => cluster-local identities
-   0x00010000 - 0x00FFFFFF (2^16        to 2^24 - 1)        => identities for remote clusters
-   0x01000000 - 0x0100FFFF (2^24        to 2^24 + 2^16 - 1) => identities for CIDRs (node-local)
```

Comment:
Is this line no longer true? I presume we specified it only includes the 64K entries with the `0x0100` prefix for a reason (maybe there's context via git logs / gh pr logs for the relevant PR).

Reply:
I didn't change the allocator (at least, I don't *think* I did). I just realized the docs were wrong corrected them. Lemme dig in a bit.